README.md

```diff
@@ -240,12 +251,10 @@ LanguageRegion_MetricUnits: no                   # Set Metric Units
 ```
 
 ```
-LaunchServices_Enabled: no                       # Enable Launch Services configuration
 LaunchServices_LSQuarantine: yes                 # no = disable the “Are you sure you want to open this application?” dialog
 ```
 
 ```
-LoginWindow_Enabled: no                          # Enable Launch Services configuration
 LoginWindow_DisableConsoleAccess: no             # Toggle login for hidden user '>Console'
 LoginWindow_GuestEnabled: no                     # Toggle guest login
 LoginWindow_LoginwindowText: ""                  # Text message to show in the login screen
```

Comment:
`LoginWindow_LoginwindowText` should be `LoginWindow_LoginWindowText`with an uppercase W

Reply:
done